packages/protocell/protocell.1.0.0/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+homepage: "https://github.com/martinslota/protocell/"
+dev-repo: "git+https://github.com/martinslota/protocell.git"
+bug-reports: "https://github.com/martinslota/protocell/issues"
+doc: "https://martinslota.github.io/protocell/"
+license: "Apache-2.0"
+maintainer: "Martin Slota <martin.slota@keemail.me>"
+build: [["dune" "build" "-p" name "-j" jobs]]
+run-test: [["dune" "runtest" "-p" name "-j" jobs]]
+depexts: [
+  ["libprotoc-dev" "protobuf-compiler"] {os-distribution = "debian"}
+  ["libprotoc-devel" "protobuf-compiler"] {os-distribution = "mageia"}
+  ["libprotoc-dev" "protobuf-compiler"] {os-distribution = "ubuntu"}
+  ["protobuf-devel" "protobuf-compiler"] {os-distribution = "centos"}
+  ["protobuf-devel" "protobuf-compiler"] {os-distribution = "fedora"}
+  ["protobuf-devel" "protobuf-compiler"] {os-distribution = "rhel"}
+  ["protobuf-dev" "protobuf"] {os-distribution = "alpine"}
+  ["protobuf-devel"] {os-family = "suse"}
+  ["protobuf"] {os = "freebsd"}
+  ["protobuf"] {os = "macos" & os-distribution = "homebrew"}
+]
+depends: [
+  "alcotest" {with-test}
+  "base" {>= "v0.10"}
+  "core" {with-test & >= "v0.10"}
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
Dune cannot be tagged build dependency due to lack of forward-compatibility. According to your dune-project file you need at least 1.11